### PR TITLE
Add fixes for 'Copy Model' PR

### PR DESCRIPTION
**Pull request by jens-br**
This PF includes some minor fixes for the Copy Model PR.

### CODE REVIEW

File: src/core/model/model.ts [hunk 1]
```diff
@@ -190,7 +190,7 @@ export class Model extends Config {
   }
 
   /**
-   * Check if the model has parameter.
+   * Check if the model has parameters.
```
Comment (babsey) [inline src/core/model/model.ts:193]: Actually it check if the requested parameter is provided in params.

Comment (babsey) [inline src/core/model/model.ts]: It actually checks if the request parameter id is provided.

**Review by babsey — CHANGES_REQUESTED** [review]
It looks good to me but a minor suggestion in the code.

Furthermore, can you change multiple codes (lines 28 and 56) in `core/simuation/simulationCodes/NEST3.2.code`: `{{ existing }}` and `{{ new }}` to `{{ existingModelId }}` and `{{ newModelId }}`, respectively?

File: src/core/model/model.ts (hunk 1 above)
Comment (jens-br) [reply src/core/model/model.ts]: Ok, I adressed this in the commits, too.

**Comment by jens-br** [comment]
I adressed these suggestions and the one from the discussion in https://github.com/nest-desktop/nest-desktop/pull/331#pullrequestreview-937323614 in the new commits.

**Review by babsey — APPROVED** [review]
LGTM!

**Pull request merged**